README.md

```diff
@@ -7,13 +7,13 @@ drone-wall
 
 ### Deployment
 
-    $ docker pull scottwferg/drone-wall
-    $ docker run -p 3000:3000 -e API_SCHEME=$API_SCHEME -e API_DOMAIN=$API_DOMAIN \
-        -e API_TOKEN=$API_TOKEN -e API_PORT=$API_PORT scottwferg/drone-wall
+    $ docker pull scottwferg/drone-wall:2.1
+    $ docker run -p 3000:8080 -e API_SCHEME=$API_SCHEME -e API_DOMAIN=$API_DOMAIN \
+        -e API_TOKEN=$API_TOKEN -e API_PORT=$API_PORT -e WALL_PORT=8080 scottwferg/drone-wall:2.1
 
 Drone wall exposes port `3000`. You can map this to whatever you like.
 
 The `API_` variables are required to configure the route that's hit to find new Drone builds.  
```

Comment:
I propose this whole block be re-written to something like this for clarity:

| ENV        | Meaning                                                                                             |
|------------|-----------------------------------------------------------------------------------------------------|
| API_SCHEME | The URL scheme the Drone API is using. (HTTPS or HTTP)                                              |
| API_DOMAIN | The URL domain the Drone API is using.                                                              |
| API_PORT   | The URL port the Drone API is using. (default 443 for HTTPS and 80 for HTTP)                         |
| API_TOKEN  | The access token that will authenticate you with the Drone API. (found in your Drone user settings) |
| WALL_PORT  | The port drone-wall is started and bound to.                                                        |

code:
```
| ENV        | Meaning                                                                                             |
|------------|-----------------------------------------------------------------------------------------------------|
| API_SCHEME | The URL scheme the Drone API is using. (HTTPS or HTTP)                                              |
| API_DOMAIN | The URL domain the Drone API is using.                                                              |
| API_PORT   | The URL port the Drone API is using. (default 443 for HTTPS and 80 for HTTP)                         |
| API_TOKEN  | The access token that will authenticate you with the Drone API. (found in your Drone user settings). |
| WALL_PORT  | The port drone-wall is started and bound to.                                                        |
```